CONTRIBUTING.md

```diff
@@ -196,15 +170,16 @@ PR descriptions should describe the motivation and context of the code changes i
 both for the reviewer and also for future developers.
 If there's a JIRA ticket or Github issue, the PR should be linked to the ticket/issue to provide that context.
 
-Travis runs the tests in `scripts/run_tests.sh`, which gives a convenient one-line invocation for testing.
-
 As it can be easy to forget to verify these prior to pushing, it's possible to use [git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to enforce compliance during normal workflows.
 Consider editing `.git/hooks/pre-commit` or `.git/hooks/pre-push` (or adding them and marking them as executable: `chmod +x <file>`).
 For example, you could set your local `.git/hooks/pre-commit` to be
 ```shell
-scripts/run_tests.sh
+python $REPO_DIR/scripts/validate_version_bump.py                        || exit 1;
+flake8 $REPO_DIR/src                                                          || exit 1;
+derp $REPO_DIR/src $REPO_DIR/src/citrine/__version__.py                       || exit 1;
+pytest --cov=src --cov-report term-missing:skip-covered --cov-config=tox.ini --no-cov-on-fail --cov-fail-under=100  tests     
```

Comment:
What is the value is removing this?

Reply:
Same as above. If we leave the script, I guess we can revert this doc, although I like people knowing exactly what's being run. For example, it wasn't until I created the `release/3.0` branch that I learned about `derp`.